### ISSUE TITLE
Hur skapar jag en HTTP-token?

### ISSUE THREAD

**Issue by Angamanga** [ss223ck/1dv450_ss223ck#12]
Jag lyckas inte skapa ett event utan får "HTTP Token: Access denied" när jag försöker skapa ett event med min api-key. Hur gör jag för att skapa en Http token?

**Comment by ss223ck** [comment]
Om du ska skapa ett event måste du skicka med en api-nyckel både som header och i body:n av förfrågan. Har du testat att skapa med hjälp av postman-filen? Det krävs att du byter ut api-nyckeln som redan står i postman-filen mot någon i databasen

**Comment by Angamanga** [comment]
Ja, jag har provat att använda en api-nyckel som jag skapat själv, både i bodyn och som header och det funkar inte. Skulle du kunna dubbelkolla att allt är korrekt i postman-filen?

**Comment by ss223ck** [comment]
Hej ledsen att jag svarar så segt. Det är något fel på min restrict_access. Om du kommenterar bort before_filter i events kontrollen så ska det fungera. Vet inte varför den helt plötsligt slutat att fungera

**Comment by ss223ck** [comment]
Hade visst fel. Det är när man skickar in en felaktig nyckel i headern som det inte fungerar. Det måste vara något fel på din api-nyckel. Har du kört rake db:seed och sedan plockat ut ett värde ur api-nyckeln som blir automatgenererad?

**Comment by Angamanga** [comment]
Jag har skapat en api-nyckel och använt den i postman-filen. När jag tittar hur du skrivit i postman-filen ser det ut som om du använt två olika nycklar i bodyn och i hedern för "Post create event". Ska den i headern vara encodad på något vis?

**Comment by ss223ck** [comment]
API-nyckel automatgenereras när man skapar en användare. Använder något speciellt gem för att det ska bli rätt. Innan API-nyckeln ska det stå "Token token=" i headern. Om du får meddelandet att HTTP Token: Access denied så är det fel på headern eller nyckeln du skickar med. Vet inte direkt vad du menar med encoded. Hur har du skapat api-nyckeln?

**Comment by Angamanga** [comment]
Nu fick jag det att fungera, det hade smugit sig in ett mellanrum for mycket efter token=. Med encoded menar jag exempelvis base64 som ibland används.
